pkg/stack/service.go

```diff
@@ -15,7 +16,10 @@ type service struct {
 }
 
 func (s service) Find(name string) (*model.Stack, error) {
```

Comment:
In `FindAll` we return a copy of the stacks `[]model.Stack`. I think we should also return `model.Stack` instead of `*model.Stack` here